devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install nginx
+  yum: name=nginx state=present
+
+- name: http service state
+  service: name=nginx state=started enabled=yes
+...
```

Comment:
I started adding that via ansible's yml syntax for closing the script.

[source](http://docs.ansible.com/ansible/latest/YAMLSyntax.html)